elements/pfe-jump-links/README.md

```diff
@@ -0,0 +1,109 @@
+# PatternFly Element | Jump links element
+
+
+## Usage
+This component is focused on reading content inside the <pfe-jump-links-panel> and updating a sidebar or horizontal nav with to indicate the section of the document currently being viewed. As a developer, you have the choice of whether you bring your own markup to the nav, or have the nav build a DOM tree for you. Please note the shape of the DOM tree below. In order to support sub-section highlighting jump links must be very opinionated about the DOM tree.
+
+Inside of <pfe-jump-links-panel> add the class .pfe-jump-links-panel__section to tell the component to watch for when that section crosses the scroll threshold. If this section has sub-sections you add the class .has-sub-section to the section parent (i.e. section 8) and add .sub-section to the child sections (i.e. 8.1). Make sure to add an id to each section, this id will match to the href attribute of the corresponding <a> tag.
+
+## Wiring up the nav
+The panel and nav are wired up by a scrolltarget and id. On the panel add an attribute scrolltarget="foo". This will correspond to the nav's #id. Add the corresponding id to your nav like so id="foo". The last step is to match the <a> tag's href attribute to specific sections (just like we would with same page anchor links). See below for a simple example with three sections where section two has two sub-sections:
+
+```html
+<pfe-jump-links-nav id="jumplinks1" default>
```

Comment:
@Djfaucette its not too late to ask a big-picture question right? 
why is the component folder and nickname `pfe-jump-links` but the HTML tag is `pfe-jump-links-nav`

cc @kylebuch8